Base.rte/Devices/Weapons/Shotgun/Shotgun.lua

```diff
@@ -0,0 +1,49 @@
+function Create(self)
+	self.pullTimer = Timer();
+	self.loaded = false;
+	self.rotFactor = math.pi;
+end
+function Update(self)
+	local parent;
+	local actor = self:GetRootParent();
+	if actor and IsAHuman(actor) then
+		parent = ToAHuman(actor);
+	end
+	if self.FiredFrame then
+		self.shell = CreateMOSParticle("Shell");
```

Comment:
It's generally a good idea to specify the rte when Creating things. Under the hood, if you don't, it looks through every RTE til it finds the appropriately named thing, which is obviously slower. In this case it'll start with Base.rte and find it right away, but still, best practices and all that.

Also, I guess this script is just for fancy shell ejection and reloading?